CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 - **CPU/Memory scaler**: Add support for scale to zero if there are multiple triggers([#4269](https://github.com/kedacore/keda/issues/4269))
 - **Redis Scalers**: Allow scaling using redis stream length ([#4277](https://github.com/kedacore/keda/issues/4277))
+- **Redis Scalers**: Allow scaling using consumer group lag ([#4592](https://github.com/kedacore/keda/issues/4592))
```

Comment:
Let us change this to https://github.com/kedacore/keda/issues/3127 as per @zroubalik's comment